src/main/java/pro/trevor/tankgame/rule/definition/actions/DiceSet.java

```diff
@@ -1,8 +1,9 @@
 package pro.trevor.tankgame.rule.definition.actions;
 
+import pro.trevor.tankgame.util.Random;
```

Comment:
`public T roll(Random random) {` should probably take an IRandom or you could remove that method since we didn't end up using it.